### ISSUE TITLE
using RedisProxy gives error - Redis::FutureNotReady

### ISSUE THREAD

**Issue by loicginoux** [kickstarter/rack-attack#462]
Hi, 
I used the RedisStoreProxy but run into the issue describe here:
https://github.com/kickstarter/rack-attack/issues/213

Trying to fix the issue, I switched to the more recent RedisProxy, as mentioned in the issue above :

```
Rack::Attack.cache.store   = Rack::Attack::StoreProxy::RedisProxy.new(Redis.current)
```

The error is now the following:

```
Redis::FutureNotReady
Value will be available once the pipeline executes.

vendor/bundle/ruby/2.6.0/gems/redis-3.3.5/lib/redis/pipeline.rb:129:in `value&#x27;
vendor/bundle/ruby/2.6.0/gems/rack-attack-6.2.1/lib/rack/attack/store_proxy/redis_proxy.rb:43:in `increment&#x27;
```

Not sure if this is an issue to with this Rack attack gem or my current setup and redis versions. Maybe others run in the same problem...

Versions:
Ruby: 2.6.0
Rack attack: 6.2.1
Redis: 3.3.5

**Comment by av-pinzur** [comment]
We're seeing this also. Thanks in advance for your attention, and thanks for all you do!

**Comment by grzuy** [comment]
Hi @loicginoux, 

Thanks for reporting!

This was already reported (https://github.com/kickstarter/rack-attack/issues/437) and fixed (https://github.com/kickstarter/rack-attack/pull/445) but **not released**. Doing so now...

**Comment by grzuy** [comment]
Fixed in `v6.2.2`.
